test/tools/d_do_test.d

```diff
@@ -933,17 +933,17 @@ bool compareOutput(string output, string refoutput, const ref EnvData envData)
         foreach (const chunk; refparts[0].splitter('|'))
         {
             // ( <predicate> , "=", <content> )
-            const conditional = chunk.findSplit("=");
+            const searchResult = chunk.findSplit("=");
```

Comment:
This seems unrelated?

Reply:
Yes, but it was a small change and to me a better name. Happy to change back if you're not convinced.

Reply:
No strong feelings about either name but PRs / commits should not contain unrelated changes.

Reply:
Hmm, it's related to this commit as conditional only made sense with `if (!conditional)` which was removed here.
Anyhow, I split the commit into two.